app/src/main/res/layout-sw600dp-port/topic_preview_fragment.xml

```diff
@@ -0,0 +1,277 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is it possible to consolidate this & the other topic preview fragments into a single layout file? We have an active effort to do that with many of our other layouts, so it seems like duplicating this new one is the wrong direction since it just pushes back that work.